spec/katello_certs_check_spec.rb

```diff
@@ -5,35 +5,39 @@
 # badkey passphrase is 'foreman'
 
 describe 'katello-certs-check' do
-  let(:command) { File.join(__dir__, '..', 'bin', 'katello-certs-check') }
-  let(:directory) { File.join(FIXTURE_DIR, 'katello-certs-check') }
-  let(:ca) { File.join(directory, 'cacert.crt') }
-  let(:key) { File.join(directory, 'foreman.example.com.key') }
-  let(:cert) { File.join(directory, 'foreman.example.com.crt') }
-  let(:badkey) { File.join(directory, 'key_pass.key') }
-
   def fixture(filename)
     File.read(File.join(directory, filename)).gsub('|COMMAND|', command)
   end
 
-  it 'without parameters' do
-    stdout, stderr, status = Open3.capture3(command)
-    expect(stderr).to eq fixture('missing-parameter.txt')
-    expect(stdout).to eq ''
-    expect(status.exitstatus).to eq 1
-  end
+  let(:command) { File.join(__dir__, '..', 'bin', 'katello-certs-check') }
+  let(:directory) { File.join(FIXTURE_DIR, 'katello-certs-check') }
+  let(:certs_directory) { File.join(directory, 'certs') }
+  let(:ca) { File.join(certs_directory, 'ca.crt') }
```

Comment:
Is it needed to move these from where they were? Moving things around makes a diff harder to read and thus more work for a reviewer.

Reply:
Do you mean the certificates themselves or do you mean these lines in the test? In both cases,  For the former, yes. For the latter, strictly needed? No. Does it help to draw a clearer line for the context? Yes. I am also looking ahead at PRs that I have that will all need to make this structural change. And it being easier to do so here, and provide additional context with the change.

Reply:
I dunno if it helps, but Github provides a method to remove whitespace changes from the diff when reviewing, e.g.

https://github.com/theforeman/foreman-installer/pull/618/files?diff=unified&w=1